File: week3 - Application Development 1 CRUD with Cassandra/crud-python/insert.py
#!/usr/bin/env python3
from db_connection import Connection
import uuid

# this is a insert statement in python
try:
    connection = Connection()
    output = connection.session.execute(
        "INSERT INTO killrvideo.user_credentials (email, password, userid) VALUES (%s, %s, %s)", 
        ('cv@datastax.com', '3@$tC0@$tC@ss@ndr@', uuid.UUID('{55555555-5555-5555-5555-555555555555}'))
    )
except:
    print('Failure')
else:
    print('Success')
finally:
    connection.close()

File: week3 - Application Development 1 CRUD with Cassandra/crud-nodejs/insert.js
const connection = require('./db_connection')
const Uuid = require('cassandra-driver').types.Uuid;

// this is an insert statement in nodejs
const insert = 'INSERT INTO killrvideo.user_credentials (email, password, userid) VALUES (?,?,?)';
const params = ['cv@datastax.com', '3@$tC0@$tC@ss@ndr@', Uuid.fromString('55555555-5555-5555-5555-555555555555')];
connection.client.execute(insert, params)
.then(function (result){
    console.log('Success')
    connection.client.shutdown()
})
.catch(function (error){
     console.log(error.message)
     connection.client.shutdown()
});